src/Example.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from "react";
+import { determineWeather } from "./util/determineWeather";
+import { Card } from "./components/Card";
+import { Temperature } from "./components/Temperature";
+
+export const Example = () => {
+  const [card1Temperature, setCard1Temperature] = useState(110);
```

Comment:
When we call useState we always get 2 things back, the piece of state, and the setter to set that piece of state. We can not edit the piece of state directly, we need to tell react to update the DOM so the user sees the changed value.

https://react.dev/learn/updating-objects-in-state